File: C++/1684. Count the Number of Consistent Strings.cpp
class Solution {
public:
    int countConsistentStrings(string s, vector<string>& w) {
        set<int>a;
        int i,j,k=0;
        for(char c:s){
            a.insert(c);
        }
        for(i=0;i<w.size();i++){
            for(j=0;j<w[i].size();j++){
                if(!a.contains(w[i][j])){
                    k++;
                    break;
                }
            }
        }
        return(w.size()-k);
    }
};

File: JAVA/1684. Count the Number of Consistent Strings.java
class Solution {
    public int countConsistentStrings(String s, String[] w) {
        HashSet<Character> a=new HashSet<>();
        int i,j,k=0;
        for(i=0;i<s.length();i++){
            a.add(s.charAt(i));
        }
        for(i=0;i<w.length;i++){
            for(j=0;j<w[i].length();j++){
                if(!a.contains(w[i].charAt(j))){
                    k++;
                    break;
                }
            }
        }
        return (w.length-k);
    }
}